src/views/UserProfileView.vue

```diff
@@ -322,7 +343,6 @@
         }
         .profile__image--edit{
           &>div{
-            position: absolute;
             top: 20%;
```

Comment:
this is unneeded if the div is position:static

Reply:
Change avatar text was not fully visible otherwise.